internal/credential/vault/repository_credential_store_test.go

```diff
@@ -27,11 +29,17 @@ func TestRepository_CreateCredentialStoreResource(t *testing.T) {
 		assert, require := assert.New(t), require.New(t)
 		ctx := context.Background()
 		kms := kms.TestKms(t, conn, wrapper)
-		repo, err := NewRepository(rw, rw, kms)
+		sche := scheduler.TestScheduler(t, conn, wrapper)
+		repo, err := NewRepository(rw, rw, kms, sche)
 		require.NoError(err)
 		require.NotNil(repo)
 		_, prj := iam.TestScopes(t, iam.TestRepo(t, conn, wrapper))
 
+		trj, err := NewTokenRenewalJob(rw, rw, kms, hclog.L())
+		require.NoError(err)
+		err = sche.RegisterJob(context.Background(), trj)
+		require.NoError(err)
+
```

Comment:
Why add these to this test?

Reply:
`repo.CreateCredentialStore` attempts to update the next runtime of the TokenRenewalJob and errors out if the job does not exist.